lib/enkf/local_config.cpp

```diff
@@ -172,6 +172,9 @@ void local_config_free(local_config_type * local_config) {
 }
 
 local_ministep_type * local_config_alloc_ministep( local_config_type * local_config , const char * key, analysis_module_type* analysis_module) {
+  if (hash_has_key(local_config->ministep_storage, key))
```

Comment:
are we sure that this wont affect other users?

Reply:
Well -  current master leads to a use-after-free situation; so this fix I am confident about.

Reply:
Very well, but it should either exit with a meaningful error or it should be dealt with here as well:
https://github.com/joakim-hove/libres/blob/388d0b29eb86a6745d5847f486c324ff0544dc39/lib/enkf/enkf_main.cpp#L1563

Reply:
I don't really understand what you request here. The python part - which is the user accessible will raise `KeyError()` - when it comes to the C++ function you link to that is called exactly once at object construction/initialization. I can of course add a 
```C++
if (!ministep) 
   throw std::logic_error("Failed to create ALL_ACTIVE ministep");
```
- or maybe I misunderstood your request?

Reply:
That is what I meant :) As we are changing the behavior of the function, it should be reflected wherever the function is used.

Reply:
ok